### ISSUE TITLE
plot() outputs require white background (or white lines?) for dark mode website

### ISSUE THREAD

**Issue by avallecam** [epiverse-trace/epiversetheme#21]
currently, the plot() outputs are lost in the background. possibly the standard suggested action item would be to try to keep the default white background.
![image](https://github.com/user-attachments/assets/401dc806-4c86-4bd3-bc5b-69ff78afd3e3)

**Comment by Bisaloo** [comment]
Can you share the link to the page where you are observing this please?

**Comment by avallecam** [comment]
> Can you share the link to the page where you are observing this please?

yes, here it is https://epiverse-trace.github.io/epiparameter/articles/data_from_epireview.html

**Comment by Bisaloo** [comment]
I see the same behaviour in pkgdown: https://pkgdown.r-lib.org/articles/test/rendering.html

Could you please open a new issue there (after checking if it hasn't been reported already)? I'm curious to see if they consider it's a bug / something that needs to be changed upstream or if they prefer leave it to end users.

**Comment by avallecam** [comment]
> Could you please open a new issue there (after checking if it hasn't been reported already)?

sure!

**Comment by avallecam** [comment]
> I'm curious to see if they consider it's a bug / something that needs to be changed upstream or if they prefer leave it to end users.

they prefer to leave this to end users using `par(bg = 'white')`.

So, should we add this parameter to all vignettes expecting that it may generate base R plots? This would give it an output similar to ggplot2 visualizations 

(screenshot from PR preview adding this to the testing vignette)
![image](https://github.com/user-attachments/assets/ce2a023d-afff-4c03-9055-93c6111e84f7)

**Comment by avallecam** [comment]
update: maybe we can wait until the discussion on their side gets to a conclusion to plan a next step.
